packages/flame/lib/src/components/component.dart

```diff
@@ -44,6 +44,14 @@ class Component {
   Component? get parent => _parent;
   Component? _parent;
 
+  /// Returns the first child that matches the given type [T].
```

Comment:
```suggestion
  /// Returns the first child that matches the given type [T].
  ///
  /// As opposed to [children.whereType<T>().first] this method returns null
  /// instead of a [StateError] when no matching children are found.
```